src/scripts/release.sh

```diff
@@ -23,20 +23,20 @@ fi
 
 ORIGIN_URL=$(git remote get-url origin)
 UPSTREAM="anz-bank"
-UPSTREAM_URL=$(echo "$ORIGIN_URL" | sed -n "s/\\(.*\\/\\/github.com\\/\\)\\(..*\\)\\(\\/..*.git\\)/\\1$UPSTREAM\\3/p")
+UPSTREAM_URL=$(echo "$ORIGIN_URL" | sed -E 's|(.*//github.com/)(.*)(/.*\.git)|\1'"${UPSTREAM}"'\3|')
```

Comment:
You may as well make this $UPSTREAM and not ${UPSTREAM} as the rest of the script uses the first form. I always use the second and I type it without thinking which is why I said it last time - it makes no difference most of the time.

Also, did `+` not work in the regex? I recall having issues with it in the past, but if you're not using `.+`, you should use `..*` as you were before.

Reply:
Argh - sorry, just sloppy on my side. Something else didn't work blablabla
Done.